mne/channels/layout.py

```diff
@@ -1000,6 +1000,80 @@ def _merge_grad_data(data, method='rms'):
     return data.reshape(data.shape[:1] + orig_shape[1:])
 
 
+def _combine_meg_grads(inst, method='rms'):
+    """Combine pairs of gradiometer channels."""
+    # TODO:
+    # - modify coil_type and kind to a combined gradiometer
+    # - modify units depending on method
+    # - need to handle noise_cov? projections?
+    # - currently assumes inst only contains 'grads'
```

Comment:
To start you could do `inst.copy().pick(...)`, but in theory this should already live in `as_type` in some reasonable way